deploy/storage/bigquery.version

```diff
@@ -1 +1 @@
-datcom-store.dc_kg_2023_03_20_23_14_35
\ No newline at end of file
+datcom-store.dc_kg_2023_04_04_00_35_41
```

Comment:
Was there a newer one - dc_kg_2023_04_10_23_19_57 - that completed last night?

https://groups.google.com/a/google.com/g/datacommons/c/INNVIJaDNWM

Reply:
updated!